frontend/packages/dev-console/integration-tests/support/step-definitions/addFlow/create-from-container-image.ts

```diff
@@ -72,6 +72,11 @@ When('user selects the {string} from Runtime Icon dropdown', (runTimeIcon: strin
   containerImagePage.selectRunTimeIcon(runTimeIcon);
 });
 
+When('user enters Name as {string}', (name: string) => {
+  console.log('name: ', name);
```

Comment:
@Lucifergene why do we need this log statement here?

Reply:
@divyanshiGupta Its not required. Have removed the line